apiMethods/articlesMethods.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Created by camel on 2016-01-06.
+ */
+var models = require('../models');
+
+module.exports = (function () {
+    return {
+        getSingleArticle: function (req, res) {
+            models.articles.getSingleArticle(req.params.id)
+                .then(function (article) {
+                    if (req.params.id != parseInt(req.params.id, 10)){
+                        res.send({status: 'error', error: 'Id is not an integer.'});
+                    }
+                    else if (!article){
+                        res.send({status: 'error', error: 'Article not found.'});
+                    }
+                    else {
+                        res.json(article);
+                    }
+                });
+        },
+        getAllArticle: function (req, res) {
+            models.articles.getArticles()
+                .then(function (articles) {
+                    if (!articles) {
+                        console.log(res.send({error: 'There is no articles.'}));
```

Comment:
tutaj to jest bez sensu bo chyba jak nie ma artykulow to powinno po prostu zwrocic pusta tablice i tyle